test/e2e/e2e.sh

```diff
@@ -35,6 +72,36 @@ if [ $gpu_test_1 != 2 ]; then
     exit 1
 fi
 
+gpu_test1_pod0_ctr0_logs=$(kubectl logs -n gpu-test1 pod0 -c ctr0)
```

Comment:
I'm very open to suggestions for ways to make this less of a copy-paste nightmare.